src/commonComponents/link/Link.js

```diff
@@ -0,0 +1,28 @@
+// Libraries
+import PropTypes from 'prop-types'
+
+
+function Link(props){
+
+    const { category, className, onClickHandler } = props;
+
+    function handleClick(){
+        onClickHandler(category);
+    }
+
+    return (
+        <a key={category} href={`#${category}`} className={className} onClick={handleClick}>{category} </a>
```

Comment:
use _startCase while showing {category}, because what if we get a lowerCase category string from API, we can't be sure about that